exporter/otlpexporter/go.mod

```diff
@@ -21,6 +21,8 @@ require (
 	google.golang.org/protobuf v1.31.0
 )
 
+require go.opentelemetry.io/otel/sdk/metric v1.20.0 // indirect
```

Comment:
Any reason why this is in a separate statement instead of together with the rest of the `// indirect` deps?

Reply:
Nope... this was an issue with the merge. Fixed